.github/workflows/publish.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  version:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.BOT_PAT }} # Hack: A PAT from an admin account to override our branch protections
+
+      - name: Setup git
+        run: |
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+          git config user.name "$GITHUB_ACTOR"
+
+      - name: Version
+        env:
+          PR_TITLE: ${{ toJson(github.event.pull_request.title) }}
+        run: |
+          /bin/bash ./bin/npm-version.sh "$PR_TITLE"
+
+      # GH Actions don't infinite loop:
+      # https://github.community/t5/GitHub-Actions/How-to-push-to-protected-branches-in-a-GitHub-Action/m-p/36156/highlight/true#M2461
+      # Except when using a PAT (see 'with token' above in checkout step):
+      # https://github.community/t5/GitHub-Actions/How-to-push-to-protected-branches-in-a-GitHub-Action/m-p/36156/highlight/true#M2461
+      - name: Push
+        run: git push --follow-tags
+
+  publish:
+    needs: version
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '10.x'
+
+    - name: Publish
+      run: npm publish
```

Comment:
Is there more to this command?

Is there anything else that needs to occur here? I'm guessing the obvious is we're going to need to give access to our NPM publishing.

Reply:
i think we will at least need to do `npm login` prior to that. i can't think of any other steps besides that tho

Reply:
We can do it with tokens instead